README.md

```diff
@@ -54,20 +54,54 @@ This setup repository comes with a [docker compose](https://github.com/Aam-Digit
 4. Set the `VIRTUAL_HOST`and`LETSENCRYPT_HOST` as environment variables on new docker containers to define under which URL they should be reachable
 
 # User management in Keycloak
-The system supports the [Keycloak](https://www.keycloak.org/) identity management system.
-This is optional but allows to enable further features like password reset in the application.
-To enable this follow the following steps.
+The system uses the [Keycloak](https://www.keycloak.org/) identity management system.
+All the required configuration can be found in the `keycloak` folder.
 
 To start the required docker containers execute the following (this is only needed once on a server, you can skip these steps if you just want to add another Aam Digital instance to an existing keycloak server):
 1. Open the file `keycloak/.env`
 2. Set the password variables to secure passwords and assign valid urls for the Keycloak and [account backend](https://github.com/Aam-Digital/account-backend) (without `https://`)
 3. Open `keykloak/realm_config.json` and add the required settings for you email server to enable Keycloak to send emails in your name
-4. Open `keykloak/create_realm.sh` and set the `<DOMAIN>` to the general domain name of you applications (e.g. `aam-digital`)
-5. Start the required containers
+4. Start the required containers
    > cd keycloak && docker-compose up -d
 
 Once done, applications can be connected with Keycloak through the `interactive_setup.sh`.
 
+## 2-Factor-Auth
+Keycloak supports a second login factor.
+
+### Authenticator app OTP
+The only built-in second factor ist OTP using a Authenticator app.
+This can be enabled by adding the action `Configure OTP` to a user.
```

Comment:
```suggestion
This can be enabled by editing a specific user in the Keycloak "Administration Console" and adding the `Configure OTP` in the "Required user actions".
```

clarifying this for me as a noob. 

Also: Is there a way to enforce / enable by default for every user?